addons/account_tax_python/models/account_tax.py

```diff
@@ -29,13 +30,35 @@ class AccountTaxPython(models.Model):
             ":param product: product.product recordset singleton or None\n"
             ":param partner: res.partner recordset singleton or None")
 
+    @api.model
+    def _convert_record_to_local_dict(self, record):
```

Comment:
Also this method allows an arbitrary dot dict, and dot dict must be avoided at all cost because they are vulnerability prone for the safe_eval sandbox escape.

e.g. (to be put in a server action)

```py
# "Simple" example
my_partner = env['res.partner'].create({'name': 'openerp'})
env['res.partner']._fields['dbname'] = env['res.partner']._fields['name']
dotdict = env['account.tax']._convert_record_to_local_dict(my_partner)
dotdict.dbname == 'openerp'
# With a more advanced example, you can even pass arbitrary objects, not just string/integer/float/list/dict/...
env['res.partner']._fields['my_random_attribute'] = env['res.partner']._fields['comment']
env.cache.set(my_partner, my_partner._fields['my_random_attribute'], env.cr)
dotdict = env['account.tax']._convert_record_to_local_dict(my_partner)
raise UserError((dotdict.dbname, dotdict.my_random_attribute))
```

The method **must** be put as a standalone method, so it cannot be accessed through a model, and ideally it should only allow to set standard builtin types such as string, integer, float, and nothing else.